sigopt_example.m

```diff
@@ -1,40 +1,51 @@
-% 
-%
-% define dimensions
-%
-% make connection object
-conn = py.sigopt_matlab.get_connection()
+import sigopt.Connection
 
-% dimension 1
-x_bounds = py.dict(pyargs('min', 0, 'max', 1))
-x_parameter = py.dict(pyargs('name', 'x', ...
-                             'type', 'double', ...
-                             'bounds', x_bounds))
-                        
-% dimension 2                
-y_bounds = py.dict(pyargs('min', 0, 'max', 1))
-y_parameter = py.dict(pyargs('name', 'y', ...
-                             'type', 'double', ...
-                             'bounds', y_bounds))
+conn = Connection('<YOUR API TOKEN HERE>')
 
-                    
-% define parameters and name variables
-parameters = py.list({x_parameter, y_parameter})
-name = 'Franke Optimization (MATLAB)'
+experiment = conn.experiments().create(struct( ...
```

Comment:
I think `struct` is more common nowadays.  The indentation style here is definitely more Python than Matlab. 
I wonder would it make the user's life easier if we have them input a single struct like this
```
prob.name = 'Franke'
prob.var_names = ['x', 'y'];
prob.var_types = ['double', 'double'];
prob.min_bounds = [0, 0];
prob.max_bounds = [1, 1];
```
This might be less confusing than requiring 3 layers of structs, but we have to do more repackaging and checking, e.g. same dimension.